python/unitytrainers/bc/trainer.py

```diff
@@ -122,16 +122,11 @@ def get_last_reward(self):
         else:
             return 0
 
-    def increment_step(self):
+    def increment_step_and_update_last_reward(self):
```

Comment:
What is the reason behind this change ?

Reply:
Allows only a single call to the network to get both, as calls to the network are expensive, even for grabbing single variables.